journal-letters/modernized/09-chapter-7.adoc

```diff
@@ -199,12 +199,12 @@ So the God of peace keep you all in the dominion of his life,
 that therein you may reign over every hurtful thing,
 and dwell in the holy mount of safety, where the destroyer cannot come;
 upon the holy hill of Zion,
-about which he has appointed salvation for walls and bulwarks;
+around which he has appointed salvation for walls and bulwarks;
 that you may all witness peace to be within her walls, and prosperity within her palaces.
 And so every day high praises may be in your hearts and mouths unto her King,
 who is King of kings, and reigns over all, who only has immortality,
 and dwells in the light: to whom be glory and eternal praises, says my soul for evermore.
-Amen, From your friend, and one that truly loves you in the Truth.
+Amen. From your friend, and one that truly loves you in the Truth.
```

Comment:
I think we should actually pull the sentence beginning with "From your friend" down into it's own paragraph, and give it a `[.signed-section-closing]` class. That seems more correct, and seems to already have been done in the original edition -- maybe it was just an oversight that it was missed here.  So the chunk would read:

```ASCIIDOC
Amen.

[.signed-section-closing]
From your friend, and one that truly loves you in the Truth.

[.signed-section-signature]
John Burnyeat
```